Dockerfile

```
@@ -0,0 +1,28 @@
+# Use Debian as the base image
+FROM debian:bullseye
+
+# Install required packages
+RUN apt-get update && apt-get install -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)